packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewerContainer.js

```diff
@@ -93,8 +100,8 @@ function FlipbookViewerContainer({
         />
       ) : (
         <FlipbookViewer
-          defaultFrame={defaultFrame}
           enableInteractionLayer={enableInteractionLayer}
+          frame={(multiImageCloneMarkers) ? 0 : frame} // workflow.multiImageCloneMarkers = true, then we force all frames to be 0
```

Comment:
This line introduces two problems:
1. When `multiImageCloneMarkers` is configured to true, `frame` is forced to `0` here, and the flipbook controls no longer work because frame is always `0`. I tested this by loading your staging project via the PFE lab (https://www.zooniverse.org/lab/1993/workflows/3760?env=staging&pfeLab=true), I enabled "Clone markers in all frames", and then ran app-project [locally](https://local.zooniverse.org:3000/projects/kieftrav/freehand-line-multiframe/classify/workflow/3760?env=staging). While the drawn marks are cloned correctly across frames in the separate frames view, I can't play or navigate between frames in the flipbook viewer.
2. By forcing frame to `0`, it also overwrites default frame configuration that project teams can introduce per subject. See the example in FlipbookViewer's storybook [here](https://github.com/zooniverse/front-end-monorepo/blob/838543e74c41bd77dbfbd1f40c00c043c4c07e3d/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewer.stories.js#L65-L87). The SubjectViewerStore [detects](https://github.com/zooniverse/front-end-monorepo/blob/838543e74c41bd77dbfbd1f40c00c043c4c07e3d/packages/lib-classifier/src/store/SubjectViewerStore/SubjectViewerStore.js#L138-L140) if there's `metadata.default_frame` and sets `frame` accordingly, but should not be overwritten here in FlipbookViewerContainer. I suggest either creating stories or subjects in your test project with a configuration combo of default frame and clone / not clones marks.

Reply:
Reverted this back - I didn't understand the interdependencies here so this helps me understand more contextually how this prop is used further in the component rendering pipeline :)